### PR TITLE
Allow empty variables (values)

### PR DESCRIPTION
**Pull request by eexit**
Taking over #5.

Thanks,

### CODE REVIEW

**Comment by eexit** [comment]
I was wondering why not using the `silent` option instead...

**Comment by rolodato** [comment]
Hi @eexit, thanks for the PR (and for the tests! too lazy to set those up originally). I'm not sure I understood your comment, what did you mean about the `silent` option?

**Comment by eexit** [comment]
You're welcome @rolodato!

I meant that the original `dotenv` `silent` option does nothing here in your projects. Maybe, we could take advantage of this already existing `silent` option to avoid throwing exception when a variable has an empty value?

**Comment by rolodato** [comment]
I'd like to keep the behavior of the underlying `dotenv` as consistent as possible, so I don't think reusing the `silent` option for this would work. Suppose you're using the `silent` option and are expecting empty values, and you forgot to create the `.env` file altogether - this could silently fail, with no clear indication as to what the problem was. It's kind of an edge case but it would be ideal if we can preserve the intention of the original options. What do you think?

**Comment by eexit** [comment]
Yes, that's what I thought... Sounds legit to me.
If you're fine with this, then you are free to merge and release the new version :)

File: package.json [hunk 1]
```diff
@@ -19,6 +19,8 @@
     "dotenv": "^1.2.0"
   },
   "devDependencies": {
-    "assert": "^1.3.0"
+    "assert": "^1.3.0",
```
Comment (rolodato) [inline package.json]: We're using `chai.assert` now, so this can be removed

File: test/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+var chai = require('chai');
+var assert = chai.assert;
+var dotenv = require('../index.js');
+
+describe('dotenv-safe', function () {
+    it ('does not throw error when all is well', function () {
```
Comment (rolodato) [inline test/index.js]: Remove the spaces in each `it ('...` for consistency :)

**Comment by eexit** [comment]
Done.

**Comment by rolodato** [comment]
Released as 2.3.0. Thanks again!

**Pull request merged**